src/components/registration/steps.jsx

```diff
@@ -473,8 +489,21 @@ module.exports = {
         onChooseOrganization: function (name, values) {
             this.setState({otherDisabled: values.indexOf(this.organizationL10nStems.indexOf('orgChoiceOther')) === -1});
         },
-        onValidSubmit: function (formData, reset, invalidate) {
+        validateOrganizationType: function (formData) {
```

Comment:
IIRC the `required` prop does not work as we expect in this case for `CheckboxGroup`. I'm not sure if we could implement a validator in `validations.jsx` directly, but will take a look.